packages/components/src/notice/list.native.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { withDispatch, withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import Notice from './';
+import styles from './style.scss';
+
+class NoticeList extends Component {
+	constructor() {
+		super( ...arguments );
+		this.removeNotice = this.removeNotice.bind( this );
+	}
+
+	removeNotice( id ) {
+		const { removeNotice } = this.props;
+		removeNotice( id );
+	}
+
+	render() {
+		const { notices } = this.props;
+
+		return (
+			<View style={ styles.list } key={ notices.lenght }>
+				{ [ ...notices ].reverse().map( ( notice ) => (
```

Comment:
I think we could do this instead, what do you think?
```suggestion
				{ notices.reverse().map( ( notice ) => (
```

Reply:
yes, should be ok, I'll test